taxonomy/emsi/client.py

```diff
@@ -115,14 +116,41 @@ class EMSISkillsApiClient(JwtEMSIApiClient):
     Object builds an API client to make calls to get the skills from course text data.
     """
 
-    API_BASE_URL = JwtEMSIApiClient.API_BASE_URL + '/skills/versions/7.35'
+    API_BASE_URL = JwtEMSIApiClient.API_BASE_URL + '/skills/versions/8.9'
 
     def __init__(self):
         """
         Initialize base class with `emsi_open` scope.
         """
         super(EMSISkillsApiClient, self).__init__(scope='emsi_open')
 
+    @JwtEMSIApiClient.refresh_token
+    def get_skill_details(self, skill_id):
+        """
+        Query the EMSI API to get details for a particular skill.
+
+        We will be using this method to populate skill category and subcategory.
+
+        Arguments:
+             skill_id (str): Skill external id, this is the id that comes from EMSI. example: 'KS124P772D5HNCJGGQ05'
+
+        Returns:
+            (dict): A dictionary containing the skill details.
+        """
+        url = 'skills/{skill_id}'.format(
```

Comment:
here as well

Reply:
I do not think it is a good idea to use `urljoin` here since this is actually url path. I have renamed the variable to `path`.